public/style.css

```diff
@@ -5462,3 +5462,7 @@ body:not(.movingUI) .drawer-content.maximized {
     overflow: auto;
     min-height: 1.8em;
 }
+
+div[contenteditable] br {
+    display: revert;
+}
```

Comment:
Elsewhere in ST's stylesheet, all `<br>` elements are switched from their default `inline`-ish display type to `block` which seems to do really weird things in contenteditable divs and makes their line breaks only 1px tall.

`<br>` elements aren't supposed to be used for layout, technically, so it's not clear why they were set to `block` but I didn't want to switch it back globally so I did it only for contenteditables.

Reply:
Already landed in d29129f30112fea439fa458867056f3364f2db35